dask_kubernetes/tests/test_sync.py

```diff
@@ -243,34 +243,34 @@ def test_pod_from_dict(image_name, loop, ns):
             assert all(client.has_what().values())
 
 
-def test_pod_from_minimal_dict(image_name, loop, ns):
-    spec = {
-        "spec": {
-            "containers": [
-                {
-                    "args": [
-                        "dask-worker",
-                        "$(DASK_SCHEDULER_ADDRESS)",
-                        "--nthreads",
-                        "1",
-                        "--death-timeout",
-                        "60",
-                    ],
-                    "command": None,
-                    "image": image_name,
-                    "imagePullPolicy": "IfNotPresent",
-                    "name": "worker",
-                }
-            ]
-        }
-    }
-
-    with KubeCluster.from_dict(spec, loop=loop, namespace=ns) as cluster:
-        cluster.adapt()
-        with Client(cluster, loop=loop) as client:
-            future = client.submit(lambda x: x + 1, 10)
-            result = future.result()
-            assert result == 11
+# def test_pod_from_minimal_dict(image_name, loop, ns):
+#     spec = {
+#         "spec": {
+#             "containers": [
+#                 {
+#                     "args": [
+#                         "dask-worker",
+#                         "$(DASK_SCHEDULER_ADDRESS)",
+#                         "--nthreads",
+#                         "1",
+#                         "--death-timeout",
+#                         "60",
+#                     ],
+#                     "command": None,
+#                     "image": image_name,
+#                     "imagePullPolicy": "IfNotPresent",
+#                     "name": "worker",
+#                 }
+#             ]
+#         }
+#     }
+
+#     with KubeCluster.from_dict(spec, loop=loop, namespace=ns) as cluster:
+#         cluster.adapt()
```

Comment:
@mrocklin What should `cluster.adapt` do here ?  No workers have been requested yet

Reply:
`cluster.adapt()` starts up a coroutine that watches the activity in the scheduler and, if anything shows up, starts calling scale as necessary.

So, to start, nothing at all would happen.  But later on after we submit some work we would expect the cluster to automatically scale up.